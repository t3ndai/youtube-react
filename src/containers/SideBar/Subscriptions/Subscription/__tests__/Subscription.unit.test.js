import React from "react";
import { shallow } from "enzyme";
import { Subscription } from "../Subcription";

describe("Subscription", () => {
  it("renders empty Subscription", () => {
    const wrapper = shallow(<Subscription />);
    expect(wrapper).toMatchSnapshot();
  });

  it("renders Subscription with broadcast", () => {
    const wrapper = shallow(<Subscription broadcast />);
    expect(wrapper).toMatchSnapshot();
  });

  it('renders Subscription with amount of new vides = 30"', () => {
    const wrapper = shallow(<Subscription amountNewVideos={30} />);
    expect(wrapper).toMatchSnapshot();
  });
});
