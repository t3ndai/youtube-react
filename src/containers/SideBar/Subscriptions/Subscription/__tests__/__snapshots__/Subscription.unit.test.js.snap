// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Subscription renders Subscription with amount of new vides = 30" 1`] = `
<MenuItem>
  <div
    className="subscription"
  >
    <div>
      <Image
        as="img"
        avatar={true}
        src="http://via.placeholder.com/28x28"
        ui={true}
      />
      <span />
    </div>
    <span
      className="new-videos-count"
    >
      30
    </span>
  </div>
</MenuItem>
`;

exports[`Subscription renders Subscription with broadcast 1`] = `
<MenuItem>
  <div
    className="subscription"
  >
    <div>
      <Image
        as="img"
        avatar={true}
        src="http://via.placeholder.com/28x28"
        ui={true}
      />
      <span />
    </div>
  </div>
</MenuItem>
`;

exports[`Subscription renders empty Subscription 1`] = `
<MenuItem>
  <div
    className="subscription"
  >
    <div>
      <Image
        as="img"
        avatar={true}
        src="http://via.placeholder.com/28x28"
        ui={true}
      />
      <span />
    </div>
  </div>
</MenuItem>
`;
